source/config/configFlags.py

```diff
@@ -238,20 +238,20 @@ def _displayStringLabels(self):
 		}
 
 
-class AddonsAutomaticUpdate(DisplayStringStrEnum):
+class AddonUpdateCheck(DisplayStringStrEnum):
```

Comment:
this will be an API breaking change so please make an alias

Reply:
I think it's worth keeping "Automatic" as opposed to manual checking

```suggestion
class AutomaticAddonUpdateCheck(DisplayStringStrEnum):
```